tests/vcs/git/GitCommitBearTest.py

```diff
@@ -81,6 +81,11 @@ def test_check_prerequisites(self):
         finally:
             shutil.which = _shutil_which
 
+    def test_get_metadata(self):
+        self.assertEqual(
+            GitCommitBear.get_metadata().name,
+            "<Merged signature of 'run', 'check_shortlog', 'check_body'>")
```

Comment:
please test the params, I predict a test failure, `GlobalBear.get_metadata()` doesn't have any self and thus can't hold any info about this classes run definition. This only works because you only test that there *is* a run method merged in.